js/packages/components/settings/Accounts.tsx

```diff
@@ -10,7 +10,6 @@ import {
 	useThemeColor,
 	pbDateToNum,
 	closeAccountWithProgress,
-	exportAccountToFile,
```

Comment:
You should keep this, along `withInAppNotification` below

The backup feature should not be exposed on web